idb/postgres/internal/encoding/encoding_test.go

```diff
@@ -6,9 +6,6 @@ import (
 
 	"github.com/algorand/go-algorand-sdk/v2/encoding/msgpack"
 	sdk "github.com/algorand/go-algorand-sdk/v2/types"
-	"github.com/algorand/go-algorand/crypto"
-	"github.com/algorand/go-algorand/crypto/merklesignature"
-	"github.com/algorand/go-algorand/data/basics"
 	"github.com/algorand/go-algorand/ledger/ledgercore"
```

Comment:
It looks like there's just one test left here using this, `TestBaseAccountDataVersusAccountBaseDataParity`, can we delete that test? Or update it?